### PR TITLE
UCP123: Support to show markdown content in the cluster diagnosis report

### PR DESCRIPTION
**Pull request by qxhy123**

### CODE REVIEW

**Comment by breeswish** [comment]
Thanks! Could you post a screenshot for the result?

File: ui/diagnoseReportApp/components/DiagnosisTable.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { useContext, useState, useEffect } from 'react'
 import { useTranslation } from 'react-i18next'
 import { TableDef, ExpandContext, TableRowDef } from '../types'
+const ReactMarkdown = require('react-markdown')
```
Comment (baurine) [inline ui/diagnoseReportApp/components/DiagnosisTable.tsx:4]: Does it have ES module?

**Comment by qxhy123** [comment]
image

File: ui/diagnoseReportApp/components/DiagnosisTable.tsx [hunk 2]
```diff
@@ -82,7 +83,11 @@ export default function DiagnosisTable({ diagnosis }: Props) {
         </h1>
       ))}
       <h3 className="is-size-4">{t(`diagnosis.tables.title.${Title}`)}</h3>
-      {<p>{t(`diagnosis.tables.comment.${Title}`, '')}</p>}
+      {
```
Comment (baurine) [inline ui/diagnoseReportApp/components/DiagnosisTable.tsx:86]: This parenthesis is not necessary.

**Comment by breeswish** [comment]
@all-contributors please add @qxhy123   for code

**Pull request merged**